### ISSUE TITLE
permutation function

### ISSUE THREAD

**Issue by chengzhix** [yaoj90/SHINE#1]
Hi, in the python version of the code,  AES is used as the permutation function, but in the C version of the code, the permutation function is missing, is there something missing here?

**Comment by YaoJGalteland** [comment]
Yes, you are right, the c version is missing. Thank you for your feedback.
